### DISCUSSION TITLE
How to read additional information attached to a png

### DISCUSSION THREAD

**Discussion by vvyoko** [dlemstra/Magick.NET#1360] [Help]
First of all, thank you for your work

I want to read the parameters generated by stable-diffusion-webui

This is the read method of ExifLibrary
https://github.com/PetrK39/SD-EXIF-Editor/blob/8756f52dc35e3aa9849ab8375e03d7c9adae9171/SD%20EXIF%20Editor/Model/Image.cs#L30

The general message is something like this
https://github.com/PetrK39/SD-EXIF-Editor#screenshot

Sample images
https://civitai.com/gallery/242253?modelId=18978&modelVersionId=22517&infinite=false&returnUrl=%2Fmodels%2F18978%2Fbetter-hobbit-house-embedding

Finally, I wish you a happy life!

**Comment by dlemstra** [comment]
Did you solve this?

**Reply by vvyoko** [reply]
No solution
After testing 
SD-EXIF-Editor takes a bit long to read this property, 400ms+
I thought it would take as little time as exif
Now it takes too long, it's not worth adding

The second one is some images missing information 

So I decided to give up adding this feature for now

Finally, thank you for your work and have a nice life!
